Utils/src/PhotonIDisoProducer.cc

```diff
@@ -121,6 +144,29 @@ PhotonIDisoProducer::PhotonIDisoProducer(const edm::ParameterSet& iConfig):
   produces< std::vector< bool > >("fullID");
   produces< std::vector< bool > >("electronFakes");
   produces< bool >("hasGenPromptPhoton");
+  if(effArEtaLow_.size()!=7 || effArEtaHigh_.size()!=7
```

Comment:
this can just be:
`bool correctSizeVecs = !(effArEtaLow_.size()!=7 || effArEtaHigh_.size()!=7 || effArChHad_.size()!=7 || effArNuHad_.size()!=7 || effArGamma_.size()!=7 || pfNuIsoRhoCorr_EB_cut_.size()!=3 || pfNuIsoRhoCorr_EE_cut_.size()!=3 || pfGmIsoRhoCorr_EB_cut_.size()!=2 || pfGmIsoRhoCorr_EE_cut_.size()!=2);`